docsrc/conf.py

```diff
@@ -251,40 +258,3 @@ def setup(app):
     "scipy": ("https://docs.scipy.org/doc/scipy/reference/", None),
 }
 
-
```

Comment:
Why is this section being deleted? This will be needed for converting notebooks into documentation. Granted, we don't have any notebooks yet, but we will in the future. Best to keep it.